deepfence_frontend/apps/dashboard/src/features/topology/data-components/tables/PodsTable.tsx

```diff
@@ -129,10 +129,13 @@ const getAppliedFiltersCount = (searchParams: URLSearchParams) => {
     return prev + searchParams.getAll(curr).length;
   }, 0);
 };
+enum KunernetesEnum {
```

Comment:
I don't think this is any better, we can revert this

Reply:
fixed